database_change_log/updateCountryList.sql

```diff
@@ -194,7 +194,7 @@ INSERT INTO country (id, name, "in_EU", casebook_mapping) VALUES (191, 'St Pierr
 INSERT INTO country (id, name, "in_EU", casebook_mapping) VALUES (192, 'St Vincent', false, 'Saint Vincent and the Grenadines');
 INSERT INTO country (id, name, "in_EU", casebook_mapping) VALUES (193, 'Sudan', false, 'Sudan');
 INSERT INTO country (id, name, "in_EU", casebook_mapping) VALUES (194, 'Suriname', false, 'Suriname');
-INSERT INTO country (id, name, "in_EU", casebook_mapping) VALUES (195, 'Swaziland', false, 'Swaziland');
+INSERT INTO country (id, name, "in_EU", casebook_mapping) VALUES (195, 'Eswatini', false, 'Swaziland');
```

Comment:
Can you move this up so its in alphabetical order now. Also, before this is ran is prod can you do a select and compare against this list to ensure there aren't any more changes?